pkg/compliance/resource.go

```diff
@@ -0,0 +1,180 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
maybe too soon to do it, but have you consider to version the resource?
maybe create a new package: /pkg/compliance/api/v1

Reply:
That's a great point, it might be too soon. But I do need to add a version schema up top in the suite file, something like:
```
schema:
  version: ...
... other fields for suite metadata and rules ...
name: Benchmark XYZ, etc.
...
```